h2o-scala/README.md

```diff
@@ -1,2 +1,3 @@
-# H<sub>2</sub>O Scala API
+# H<sub>2</sub>O Scala API (Deprecated)
 ----
+**Deprecation warning**: This module is now deprecated and will be released in next major version `3.32.0.1`.
```

Comment:
and will be released -> and will be removed

Reply:
Thanks !